From 3f49a9cdc82f85e08fb957dde0d5d67baf4913e9 Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham. Lam" <loc.lam.px@renesas.com>
Date: Thu, 23 May 2019 15:35:08 +0700
Subject: [PATCH 140/248] arm64: dts: renesas: hihope-common: Enable bus clock
 for PCIe

This patch enables bus clock PCIE for the HiHope board.

Signed-off-by: Loc Phuoc Pham. Lam <loc.lam.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 8fb92f2..b9ee4db 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -278,3 +278,7 @@
 	clock-names = "du.0", "du.1", "du.2",
 			"dclkin.0", "dclkin.1", "dclkin.2";
 };
+
+&pcie_bus_clk {
+	clock-frequency = <100000000>;
+};
-- 
2.7.4

